app/_src/production/secure-deployment/certificates.md

```diff
@@ -157,25 +157,23 @@ The API Server HTTPS server is secured by default by autogenerated certificates.
 
 **To override autogenerated certificates.**
 
-1) Prepare certificates
-
-Generate TLS pair with a PKI of your choice and store it in PEM-encoded format in `/tmp/tls.crt`, `/tmp/tls.key`. Store the CA that was used to sign this pair in `/tmp/ca.crt`
-
-We can also use `kumactl` to generate self-signed certs:
-```sh
-kumactl generate tls-certificate \
-  --type=server \
-  --hostname=<KUMA_CP_DNS_NAME> \
-  --cert-file=/tmp/tls.crt \
-  --key-file=/tmp/tls.key
-```
-
-Since "tls.crt" is a self-signed cert, it is also a CA:
-```sh
-cp /tmp/tls.crt /tmp/ca.crt
-```
-
-2) Configure the control plane with generated certificates
+1. Prepare certificates. Generate TLS pair with a PKI of your choice and store it in PEM-encoded format in `/tmp/tls.crt`, `/tmp/tls.key`. Store the CA that was used to sign this pair in `/tmp/ca.crt`
```

Comment:
I made some formatting updates, but I'm not sure if these will render correctly since I couldn't check it out locally.

Reply:
Also, I'm wondering if this note should also be added to the previous section (Encrypted communication) as well.

Reply:
> I made some formatting updates, but I'm not sure if these will render correctly since I couldn't check it out locally.

you can always check the deployment preview https://deploy-preview-1484--kuma.netlify.app/docs/dev/production/secure-deployment/certificates/

Reply:
Thanks! I fixed the formatting